metagraph/benchmarks/benchmark_matrix.cpp

```diff
@@ -115,4 +115,98 @@ BENCHMARK_TEMPLATE(BM_BRWTQueryRows, 3000000, 100, 30, 2, false, 0)
     ->Unit(benchmark::kMillisecond)
     ->DenseRange(0, 10, 1);
 
+
+template <size_t rows_arg = 300000,
+          size_t cols_arg = 100,
+          size_t unique_arg = 10,
+          size_t arity_arg = 2,
+          bool greedy_arg = true,
+          size_t relax_arg = 2>
+static void BM_BRWTQueryColumns(benchmark::State& state) {
```

Comment:
Don't build it here, better pass a constructed one with an environment variable (have a look at boss benchmarks).

It will be much easier to benchmark things on different types of annotation. Also, you can write one benchmark for all annotation types, not only for BRWT.